back_end.py

```diff
@@ -1,21 +1,75 @@
 # -*- encoding: utf-8 -*-
-import random
+import names
+from datetime import datetime
+import logging
 from flask.ext.script import Manager
 from flask.ext.migrate import Migrate, MigrateCommand
 
 from app import app, db
-from main.models import Service, Specialist, Company
+from default_db_data import services_data
+from main.models import Service, ServiceCategory, User, UserUserActivity
+
+logging.basicConfig()
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
 
 manager = Manager(app)
 manager.add_command('db', MigrateCommand)
 migrate = Migrate(app, db)
 
 
+# create users and activities
+@manager.command
+def create_activities():
+    time = datetime.utcnow()
+    for i in range(20):
+        i = UserUserActivity(confirmed=True,
```

Comment:
You use the same variable `i` for iteration value and activity object. That's not good.